action.yml

```diff
@@ -74,7 +74,9 @@ runs:
 
         echo imageStream=${IS} >> $GITHUB_OUTPUT
         echo deploymentConfig=${DC} >> $GITHUB_OUTPUT
-        echo url=${URL} >> $GITHUB_OUTPUT
+
+        # Removes any double slashles, e.g. inputs.verification_path
+        echo url=${URL} | sed 's // / g' >> $GITHUB_OUTPUT
```

Comment:
I'm wondering if this can accidentally remove the **//** from the beginning of the URL or not (like turning https://address.com//check into https:/address.com/check)

Reply:
oh my bad, the URL does not include the schema at the beginning.